BunnymodXT/modules/ClientDLL.cpp

```diff
@@ -714,6 +714,13 @@ void ClientDLL::RegisterCVarsAndCommands()
 	EngineDevMsg("[client dll] Registering CVars.\n");
 
 	#define REG(cvar) HwDLL::GetInstance().RegisterCVar(CVars::cvar)
+
+	if (!HwDLL::GetInstance().ORIG_Cvar_FindVar(CVars::cl_righthand.GetPointer()->name))
```

Comment:
```suggestion
	if (!HwDLL::GetInstance().ORIG_Cvar_FindVar("cl_righthand"))
```
Simpler is better in this case I think

Reply:
Done.